src/main/java/ch/njol/skript/expressions/ExprRawName.java

```diff
@@ -41,79 +41,43 @@
 import java.util.Arrays;
 import java.util.List;
 
-import org.bukkit.Material;
 import org.bukkit.event.Event;
-import org.bukkit.inventory.ItemStack;
-import org.bukkit.inventory.meta.PotionMeta;
-import org.bukkit.potion.PotionData;
-import org.bukkit.potion.PotionEffectType;
 import org.eclipse.jdt.annotation.Nullable;
 
 import ch.njol.skript.Skript;
 import ch.njol.skript.aliases.ItemType;
+import ch.njol.skript.classes.Changer;
 import ch.njol.skript.doc.Description;
 import ch.njol.skript.doc.Examples;
 import ch.njol.skript.doc.Name;
 import ch.njol.skript.doc.Since;
-import ch.njol.skript.expressions.base.PropertyExpression;
 import ch.njol.skript.expressions.base.SimplePropertyExpression;
-import ch.njol.skript.lang.Expression;
-import ch.njol.skript.lang.ExpressionType;
-import ch.njol.skript.lang.SkriptParser.ParseResult;
-import ch.njol.skript.lang.util.SimpleExpression;
-import ch.njol.skript.util.PotionEffectUtils;
-import ch.njol.util.Kleenean;
 
 @Name("Raw Name")
 @Description("Raw Minecraft material name for given item. Note that this is not guaranteed to give same results on all servers.")
 @Examples("raw name of tool of player")
-@Since("unknown (2.2)")
-public class ExprRawName extends SimpleExpression<String> {
+@Since("unknown (2.2) (Estimate Mirre's edit), 2.2-dev35 (Converted to SimplePropertyExpression)")
```

Comment:
An user may don't know who is Mirre. Make it ``unknown (before 2.2)`` And again who cares about a Java thing in Skript?

Reply:
SimplePropertyExpression is an implementation detail, also. It should not be in docs for *scripters*.